src/rest_api.py

```diff
@@ -40,41 +40,17 @@ def _recommender(handler):
     external_request_id = 'None'
     recommender_started_at = time.time()
 
-    r = {'recommendation': 'failure', 'external_request_id': None}
-    # (fixme) Create decorator for metrics handling.
-    metrics_payload = {
```

Comment:
any reason for removing the prometheus payload and calls?

Reply:
It wasn't working at all, the current prometheus integration is broken !
https://sentry.devshift.net/sentry/fabric8-analytics-production/issues/272713/?query=is%3Aunresolved%20age%3A-24h

Reply:
It actually works, it does not work for SA and we need to look at it.

https://grafana.app-sre.devshift.net/d/SW0O-67Zz/code-ready-analytics-stats?orgId=1&from=now-2d&to=now

Reply:
metrics implementation should be reshaped for osd v4. I removed this because of the following sentry error.
https://sentry.devshift.net/sentry/fabric8-analytics-production/issues/272716/?query=is%3Aunresolved

Reply:
restored it. PTAL